### PR TITLE
po: Add Norwegian (bokmal) Translation

### PR DESCRIPTION
**Pull request by jeblad**
The string “Localised into seven languages” seems odd…

I can add Norwegian (nynorsk) later.

### CODE REVIEW

**Comment by mjakeman** [comment]
Thank you, much appreciated!

> The string “Localised into seven languages” seems odd…

Yeah, this is part of the changelog for the 0.2.0 release rather than the actual app. I sort of regret adding it in, because we're well over seven languages now and every new translation needs to translate it. Might have to rethink whether we translate release notes down the line (or rather, maybe only keep the past three releases?).

Anyhow, this looks good to me. Let's merge.

File: po/nb.po [hunk 1]
```diff
@@ -109,7 +109,7 @@ msgstr ""
 #. TRANSLATORS: 'Name <email@domain.com>' or 'Name https://website.example'
 #: src/exm-application.c:112
 msgid "translator-credits"
-msgstr "Kreditering av oversettere"
+msgstr "kreditering av oversettere"
```
Comment (mjakeman) [inline po/nb.po:112]: Ah actually, just noticed this. This should be substituted with your name so you are credited in the about dialog whenever someone uses the translation. So for example, you might put:

`John Doe <email@website.com>`

If you don't want to provide an email, then simply `Firstname Lastname` is fine.

**Review by mjakeman — CHANGES_REQUESTED** [review]
Just noticed an issue with `translator-credits`. One minor change, and then it's good to merge :)

**Review by mjakeman — APPROVED** [review]

**Comment by jeblad** [comment]
Sorry, forgot to check this after it was submitted! ;)

**Pull request merged**